src/core/tsi/ssl_transport_security.c

```diff
@@ -48,6 +48,7 @@
 #include <openssl/x509.h>
 #include <openssl/x509v3.h>
 
+#include "src/core/security/b64.h"
```

Comment:
tsi should only depend from tsi and <grpc/support/*.h>. If you need base64 here (not sure if you do given the previous comment), please use the openssl one.